src/dymaptic.GeoBlazor.Core/Components/Widgets/LegendWidget.cs

```diff
@@ -99,4 +112,47 @@ public class LayerInfo : MapComponent
     [Parameter]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public int[]? SublayerIds { get; set; }
-}
\ No newline at end of file
+}
+
+/// <summary>
+/// The widget legend style, sets the display style of the legend widget.
+/// <a target="_blank" href=" https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Legend.html#style">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class LegendStyle
+{
+    public LegendStyleType? Type { get; set; }
+    public LegendStyleLayout? Layout { get; set; }
+}
+
+
+/// <summary>
+/// The Legend style type.
+/// <a target="_blank" href=" https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Legend.html#style">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+[JsonConverter(typeof(EnumToKebabCaseStringConverter<LegendStyleType>))]
+public enum LegendStyleType
+{
+    Card,
```

Comment:
The XML doc warning will throw if you don't add an XML comment to _every enum member_ (ugh), so we've been using `#pragma warning disable CS1591` inside enums.